src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseHelpAndSupportSection.tsx

```diff
@@ -30,28 +30,32 @@ const ReleaseHelpAndSupportSection = ({
           caption="Find out how and why we collect, process and publish these statistics"
           headingTag="h3"
         >
-          {publication.methodology || publication.externalMethodology ? (
-            <p>
-              Read our{' '}
-              {publication.methodology &&
-                (isEditing ? (
-                  <a>{`${publication.title}: methodology`}</a>
-                ) : (
-                  <Link to={`/methodologies/${publication.methodology.id}`}>
-                    {`${publication.title}: methodology`}
-                  </Link>
-                ))}
-              {!publication.methodology &&
-                publication.externalMethodology &&
-                (isEditing ? (
-                  <a>{`${publication.title}: methodology`}</a>
-                ) : (
-                  <Link to={publication.externalMethodology.url}>
-                    {`${publication.title}: methodology`}
-                  </Link>
-                ))}{' '}
-              guidance.
-            </p>
+          {publication.methodologies.length > 0 ||
```

Comment:
Can just be `publication.methodologies.length` I think - a length of zero is falsy

Reply:
Thanks I've made this change now